serpentTools/tests/test_xsplot.py

```diff
@@ -0,0 +1,56 @@
+"""Test the xsplot reader."""
+import os
+import unittest
+import numpy
+import pickle
```

Comment:
I have mixed feelings about pickling. On one hand, your tests are super simple compared to directly comparing arrays that are hand written into the test [example: depletion tester](https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/develop/serpentTools/tests/test_depletion.py#L53).

On the otherhand, pickling is kind of closed off and potentially dangerous. When unpickling something, there is the chance to execute malicious code. The [`pickle` module documentation](https://docs.python.org/3/library/pickle.html?highlight=pickle#module-pickle) gives:

>  Warning : The pickle module is not secure against erroneous or maliciously constructed data. Never unpickle data received from an untrusted or unauthenticated source. 

We don't have any real way of knowing what's really in the pickle, aside from some of it can be used to pass this unit test. If we can come up with an open and reliable way to prove that there is nothing malicious, then pickling would be the way to go. 

Therefore for now, please remove the use of the `pickle` module. If you have ideas on improving the unit test system, please add them [here](https://github.com/CORE-GATECH-GROUP/serpent-tools/issues/103) and I will be happy to discuss alternatives